tests/PySys/pysysproject.xml

```diff
@@ -3,14 +3,18 @@
 	<requires-pysys>1.6.1</requires-pysys>
 	<requires-python>3.7.3</requires-python>
 	<property name="appHome" value="${env.PYSYS_APP_HOME}" default="${testRootDir}/.." pathMustExist="true"/>
-    <property name="tebasedir" value="${env.TEBASEDIR}" />
-    <property name="exampledir" value="${env.EXAMPLEDIR}" />
-    <property name="username" value="${env.C8YUSERNAME}" />
-    <property name="c8ypass" value="${env.C8YPASS}" />
-    <property name="tenant" value="${env.C8YTENANT}" />
-    <property name="deviceid" value="${env.C8YDEVICEID}" />
-    <property name="device" value="${env.C8YDEVICE}" />
+    <property name="tebasedir" value="${env.TEBASEDIR}" default="${testRootDir}/../.."/>
+    <property name="exampledir" value="${env.EXAMPLEDIR}" default="${testRootDir}/../../ci"/>
+    <property name="c8yurl" value="${env.C8YURL}" default=""/>
+    <property name="username" value="${env.C8YUSERNAME}" default=""/>
+    <property name="c8ypass" value="${env.C8YPASS}" default=""/>
+    <property name="tenant" value="${env.C8YTENANT}" default=""/>
+    <property name="deviceid" value="${env.C8YDEVICEID}" default=""/>
+    <property name="device" value="${env.C8YDEVICE}" default=""/>
 	<property name="platform" default='None' />
+
```

Comment:
Will this help now to run the tests on a dev system without commenting out these environment variables?

Reply:
Yeah that was the intent.

Reply:
Not absolutely sure if this is a good Idea, when a value is missing it will be hard to find out from the log what is going on. What about using "not yet configured" as default value? BTW, developers could now of course also set the default value locally to their settings.

Reply:
I've added additional validation in `EnvironmentC8y` so that the the tests will fail with proper error messages when executed with these default empty values.